src/tests/objects.rs

```diff
@@ -107,13 +107,15 @@ fn test_create_objects() {
     /* login */
     testtokn.login();
 
-    let _ = ret_or_panic!(import_object(
-        session,
-        CKO_DATA,
-        &[],
-        &byte_values,
-        &bool_values,
-    ));
+    if testtokn.dbtype != "nssdb" {
```

Comment:
This should not be necessary anymore

Reply:
The CKO_DATA object type is not posible to store in the NSS DB if I understand it right.

Reply:
Doh! Of course.. I spoke with Bob and we could actually store them, but to be able to that the nssdb.rs code will need to be changed, so nothing to do in this PR.